src/routes/sign-in/+page.svelte

```diff
@@ -3,5 +3,5 @@
 </script>
 
 <div class="flex items-center justify-center mt-16">
-	<SignIn afterSignInUrl="/admin" />
+	<SignIn path="/sign-in" />
```

Comment:
I may have found your redirect_url issue. Removing this path prop allows the sign in flow to proceed to the password and thus logs the user in.
```suggestion
	<SignIn />
```